src-tauri/src/main.rs

```diff
@@ -0,0 +1,19 @@
+// Prevents additional console window on Windows in release, DO NOT REMOVE!!
+#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]
+
+use tauri::Manager;
+
+fn main() {
+  tauri::Builder::default()
+    .setup(|app| {
+      let _window = app.get_window("main").unwrap();
+      #[cfg(debug_assertions)] // only include this code on debug builds
+      {
+        _window.open_devtools();
+        _window.close_devtools();
+      }
```

Comment:
only open the dev tooling window for debug builds of Tauri (aka. `npm run tauri dev`)